app/app.go

```diff
@@ -994,8 +994,8 @@ func NewExocoreApp(
 		evmtypes.ModuleName,     // must be before avs, since dogfood calls avs which calls this
 		exominttypes.ModuleName,
 		assetsTypes.ModuleName,
-		avsManagerTypes.ModuleName, // before dogfood, since dogfood registers itself as an AVS
 		operatorTypes.ModuleName,   // must be before delegation
+		avsManagerTypes.ModuleName, // before dogfood, since dogfood registers itself as an AVS
```

Comment:
Why was it moved after the operator module?

Reply:
We need to confirm that tasks in the avs module depend on the operator. If opearter does not exist, will it cause an error？

Reply:
Not sure, it seems like the initialization of the operator module will also depend on the AVS information in the future.  We can ignore this issue currently.